lib/routes/drivers/tests/routes.test.js

```diff
@@ -0,0 +1,167 @@
+import { resetAndMockDB } from 'utils/testUtils';
+
+const payload = {
+    firstName: 'Jhon',
+    lastName: 'Wick',
+    email: 'wick@jhon.is',
+    mobileNo: '8585959524',
+    cabTypeId: 1,
+    cabNumber: 'MH 20 9211',
+    gender: 'male'
+};
+
+const payloadForLocation = {
+    driverId: 1,
+    currentLocation: [18.52043, 73.856743],
+    status: 'active'
+};
+
+const payloadForAccepting = {
+    driverId: 1,
+    bookingAttemptId: 1
+};
+
+describe('/drivers route  tests', () => {
+    let server;
+    beforeEach(async () => {
+        server = await resetAndMockDB(async allDbs => {});
+    });
+
+    it('should return 200', async () => {
```

Comment:
Keep it a little bit more verbose. Should return 200 when proper request payload is provided.

Reply:
Done.